app/tests/actions/topics-test.js

```diff
@@ -5,8 +5,9 @@ import md5 from 'spark-md5';
 import { polyfill } from 'es6-promise';
 import axios from 'axios';
 import expect from 'expect';
-import * as actions from 'actions/topics';
-import * as types from 'types';
+import sinon from 'sinon';
```

Comment:
@ZeroCho yeah, with the previous test setup, the webpack module resolve was used, allowing us to import components relative to webpacks module paths, but it looks like this was lost with this new setup.